extensions/scarb-doc/src/docs_generation/markdown/traits.rs

```diff
@@ -14,12 +15,27 @@ pub trait TopLevelMarkdownDocItem: MarkdownDocItem + TopLevelDocItem {
         format!("{}.md", self.full_path().replace("::", "-"))
     }
 
-    fn md_ref(&self) -> String {
-        format!("[{}](./{})", self.name(), self.filename())
+    fn parent_path(&self) -> String {
+        let mut path_tree_elements: Vec<_> = self.full_path().split("::").collect();
+        path_tree_elements.pop();
+        path_tree_elements.join("::")
     }
 
-    fn generate_markdown_list_item(&self) -> String {
-        format!("- {}\n", self.md_ref())
+    fn md_ref(&self, display_parent_path: bool) -> String {
+        if display_parent_path {
+            format!(
+                "[{}  ({})](./{})",
```

Comment:
Wouldn't it actually be more readable, to just print the path? (like `hello_world::sub_module::duplicated_function_name` instead of this weird brackets `duplicated_function_name (hello_world::sub_module)`)

Reply:
You mean just printing whole path (without the name, as it's inside the path)? If so, I think it might work, will check this out and see how does it look like